index.html

```diff
@@ -0,0 +1,152 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;900&family=Lato:wght@400;700;900&family=Poppins:ital,wght@0,200;0,400;0,600;1,500&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="home.css">
+  <!-- <link rel="stylesheet" href="about.css"> -->
+  <title>Summer Concert Page</title>
+</head>
+<body>
+  <div id="overlay">
+    <div id="close-container" onclick="menuClose()">
+      <a id="closer" onclick="menuClose()">
+        <img src="./images/Icon.png" alt="Close" onclick="menuClose()">
+      </a>
+    </div>
+    <div id="menucontainer">
+        <a href="" onclick="menuClose()">Home</a>
+        <a href="" onclick="menuClose()">About</a>
+        <a href="" onclick="menuClose()">Program</a>
+        <a href="" onclick="menuClose()">Join</a>
+        <a href="" onclick="menuClose()">News</a>
+    </div>
+  </div>
+  <header class="primary-header">
+    <ul id="social-icons">
+      <li class="media-icons"><img src="./images/facebook.png" alt=""></li>
+      <li class="media-icons"><img src="./images/twitter.png" alt=""></li>
+      <li class="media-icons"><img src="./images/instagram.png" alt=""></li>
+      <li class="media-icons"><img src="./images/linkedin.png" alt=""></li>
+    </ul>
+  </header>
+  <header class="toolbar">
+    <img src="./images/summer.png" alt="" class="summer_img" >
+    <div >
+      <img src='./images/mobile_menu.png' alt="mobile_menu" id="toolbar-img" onclick= menuOpen() >
+    </div>
+    <ul id="link">
+      <li class="links"><a href="">Home</a></li>
+      <li class="links"><a href="about.html">About</a></li>
+      <li class="links"><a href="">Program</a></li>
+      <li class="links"><a href="">Join</a></li>
+      <li class="links"><a href="">News</a></li>
+    </ul>
+  </header>
+  <section >
+    <div class="introduction">
+    <h1 id="hello-text">"Hello sharing world!"</h1>
+    <h2 id="summer-text">CREATIVE SUMMER CONCERT SERIES 2020</h2>
+    <div id="details"><P class="primary-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+      nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in 
+      reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+      Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt 
+      mollit anim id est laborum.</P></div>
+      <!-- <p>Please contact us per Email for any further questions about Summer concert series</p>
+      <strong>summerconcert@hotmail.com</strong> -->
```

Comment:
- [ ] Kindly remove this commented code to make your project more professional and to improve readability 💯